clients/drcachesim/tracer/tracer.cpp

```diff
@@ -164,8 +164,9 @@ bool attached_midway;
 static bool
 bbdup_instr_counting_enabled()
 {
-    return op_trace_after_instrs.get_value() > 0 || op_trace_for_instrs.get_value() > 0 ||
-        op_retrace_every_instrs.get_value() > 0;
+    return op_trace_after_instrs.get_value() > 0 ||
+        op_L0_filter_until_instrs.get_value() > 0 ||
```

Comment:
But the L0_filter is a mode that uses tracing instrumentation, not counting instrumentation -- so this doesn't look right to me.

Reply:
I reverted this change. By the way, should the existing code use `op_trace_for_instrs` here?

Reply:
I believe -trace_for_instrs, with no other options, switches to counting mode once tracing is done, so it should be here per current code.  Now that we have a NOP mode that could be changed: maybe could add a XXX comment here.